build/repo.targets

```diff
@@ -1,9 +1,13 @@
 <Project>
   <ItemGroup>
     <Scenarios Include="-n Plaintext -o KestrelHttpServer@dev --kestrelThreadCount 1" />
+    <Scenarios Include="-n Plaintext -o KestrelHttpServer@dev --kestrelThreadCount 1 --kestrelThreadPoolDispatch false" />
```

Comment:
At the default (dispatching), Kestrel will use all cpu cores (via the ThreadPool). When set to false, Kestrel will only use kestrelThreadCount threads. The best performance when not dispatching is when kestrelThreadCount is set to the number of cpu cores (or is it cpu threads when hyperthreading?).